docs/contributing/setup.md

```diff
@@ -55,16 +55,16 @@ $ sudo ln -s `find /usr/lib64/ -type f -name "libreadline.so.7.0"` /usr/lib64/li
 With these things installed, open a shell and install `yarn` (you might need
 to `sudo` here depending on how Node was installed):
 
-```
-> npm install -g yarn@1.2.0
+```shellsession
+$ npm install -g yarn@1.2.0
```

Comment:
Shouldn't commands be prefixed with `>` to get the syntax highlighting? The below examples seem to be rendering right?

Reply:
It’s [supposed to work with a `$`](https://github.com/github/linguist/blob/master/samples/ShellSession/dollar.sh-session).

`>` is PS2, used like this:

```shellsession
$ echo "foo
> bar
> baz"
foo
bar
baz
```

Reply:
:thumbsup: just dropped a commit on top to tidy up those others